src/main/java/com/gotocompany/depot/maxcompute/converter/ConverterOrchestrator.java

```diff
@@ -0,0 +1,77 @@
+package com.gotocompany.depot.maxcompute.converter;
+
+import com.aliyun.odps.type.TypeInfo;
+import com.google.protobuf.Descriptors;
+import com.gotocompany.depot.config.MaxComputeSinkConfig;
+import com.gotocompany.depot.maxcompute.converter.payload.DurationPayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.payload.MessagePayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.payload.PayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.payload.PrimitivePayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.payload.StructPayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.payload.TimestampPayloadConverter;
+import com.gotocompany.depot.maxcompute.converter.type.DurationTypeInfoConverter;
+import com.gotocompany.depot.maxcompute.converter.type.MessageTypeInfoConverter;
+import com.gotocompany.depot.maxcompute.converter.type.PrimitiveTypeInfoConverter;
+import com.gotocompany.depot.maxcompute.converter.type.StructTypeInfoConverter;
+import com.gotocompany.depot.maxcompute.converter.type.TimestampTypeInfoConverter;
+import com.gotocompany.depot.maxcompute.converter.type.TypeInfoConverter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class ConverterOrchestrator {
```

Comment:
how about we name it ProtobufConverterOrchestrator ? Since it was located under the parent `maxcompute` package it is implied that the goal of this class is to convert proto to max compute compatible type and payload

Reply:
lets go with ProtobufConverterOrchestrator